src/constants.rs

```diff
@@ -518,6 +518,7 @@ pub const CONFIG_TEXT: &str = r##"# This is a default config file for bottom.  A
 # Override layout default widget
 #default_widget_type = "proc"
 #default_widget_count = 1
+#expanded_on_startup = false
```

Comment:
nit: add a comment on what this option does; feel free to just reuse your clap description. Also would be nice if you update `sample_configs/default_config.toml`.